src/libpriv/rpmostree-jigdo-build.c

```diff
@@ -547,6 +547,30 @@ build_objid_map_for_package (RpmOstreeCommit2JigdoContext *self,
   return TRUE;
 }
 
+/* Converts e.g. x86_64 to x86-64 (which is the current value of the RPM %{_isa}
+ * macro). I haven't yet investigated if this is accurate for other
+ * architectures.  OK wait, I just did and this whole situation is a flaming ball
+ * of horrors. I am having flashbacks to when I played Doom for the first time
+ * and had the lights off and this Cacodemon came out of nowhere...
+ *
+ * Here's where RPM maintains this currently:
+ * https://github.com/rpm-software-management/rpm/blob/d9d47e01146a5d4411691a71916b1030ac7da193/installplatform#L25
+ *
+ * I think what we need to do is add a proper API to librpm to do this
+ * inversion. For now, I'm just going to hardcode x86_64 → x86-64.
+ */
+static char *
+pkg_get_requires_isa (DnfPackage *pkg,
+                      GError    **error)
+{
+  const char *arch = dnf_package_get_arch (pkg);
+  if (g_str_equal (arch, "x86_64"))
+    return g_strdup ("x86-64");
+  return glnx_null_throw (error, "Don't know how to generate RPM %%{_isa} for arch: %s"
+                          " of package %s (see https://github.com/projectatomic/rpm-ostree/issues/XXX)",
```

Comment:
I'll file the issue and do a fixup for this after we agree with this approach.

Reply:
And let's file that issue! :)